sdcm/utils/microbenchmarking/perf_simple_query_reporter.py

```diff
@@ -0,0 +1,150 @@
+            if len(filtered_by_scylla_date.keys()) < self.collect_last_scylla_date_count:
+                if date not in filtered_by_scylla_date.keys() or median_tps > \
+                        filtered_by_scylla_date[date]["results"]["perf_simple_query_result"]["stats"][
+                            "median tps"]:
+                    filtered_by_scylla_date[date] = row
+
+            if len(filtered_results) >= self.collect_last_results_count and len(
+                    filtered_by_scylla_date.keys()) >= self.collect_last_scylla_date_count:
+                break
+
+        def make_table_line_for_render(data):
+            table_line = collections.OrderedDict()
+            table_line['test_version'] = data['results']['perf_simple_query_result']['versions'][
+                'scylla-server']
+            stats = data['results']['perf_simple_query_result']['stats']
+            for key, value in stats.items():
+                table_line[key] = value
+                if value > 0 and key != "mad tps":
+                    diff = test_stats['stats'][key] / value
+                    table_line["is_" + key + "_within_limits"] = False
+                    if diff > 1 - regression_limit:
+                        table_line["is_" + key + "_within_limits"] = True
+                    table_line[key + "_diff"] = round((diff - 1) * 100, 2)
+            return table_line
+
+        last_results_table = []
+        for result in filtered_results:
+            last_results_table.append(make_table_line_for_render(result))
+
+        scylla_date_results_table = []
+        for date, result in filtered_by_scylla_date.items():
+            scylla_date_results_table.append(make_table_line_for_render(result))
+
+        subject = "perf_simple_query Microbenchmark - Regression - %s" % test_stats['versions']['scylla-server'][
+            'run_date_time']
+        for_render = {
+            "reporter": "PerfSimpleQuery",
+            "subject": subject,
+            "testrun_id": test_details['test_id'],
+            "test_stats": test_stats,
+            "last_results_table": last_results_table,
+            "scylla_date_results_table": scylla_date_results_table,
+            "job_url": test_details['job_url'],
+            "test_version": test_stats['versions']['scylla-server'],
+            "collect_last_results_count": f"last {self.collect_last_results_count} test runs results",
+            "collect_last_scylla_date_count": f"last {self.collect_last_scylla_date_count} Scylla builds dates results",
+            "deviation_diff": deviation_diff,
+            "is_deviation_within_limits": is_deviation_within_limits,
+            "kibana_url": "https://70f106c98484448dbc4705050eb3f7e9.us-east-1.aws.found.io:9243/app/dashboards#/view/"
+                          "5f06de80-df8a-11ed-baf0-09544f843571?_g=(filters:!(),refreshInterval:(pause:!t,value:0),"
+                          "time:(from:'2023-04-16T22:00:00.000Z',to:now))"
```

Comment:
you probably don't this url hard-coded like that...

Reply:
i.e. the `from` date, might be year/month before
and also the now, should be based on the actuly run, or release date of scylla under test.

if you come back to that report, you would want to use the link to see the relevant results